### PR TITLE
Created busy indicator

### PR DESCRIPTION
**Pull request by dmitry-zezik**

### CODE REVIEW

File: src/resources.qrc [hunk 1]
```diff
@@ -159,6 +159,7 @@
         <file>qml/components/HorizontalRule.qml</file>
         <file>qml/components/NetworkStatusControl.qml</file>
 	<file>qml/components/NewGroupChatHeader.qml</file>
+	<file>qml/components/BusyControl.qml</file>
```
Comment (fpohtmeh-github) [inline src/resources.qrc:162]: wrong identation

File: src/qml/components/BusyControl.qml [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import "../theme"
+
+Item {
```
Comment (fpohtmeh-github) [inline src/qml/components/BusyControl.qml:6]: File is a 75% copy-paste of existing loader. Not acceptable

Comment (dmitry-zezik) [reply src/qml/components/BusyControl.qml:6]: For C++ ? True, but not for qml. Moreover, the functionality of different loaders is not compatible, although it seems that the code is very similar. If we try to remove duplicate code, we will only spoil the source loader code.

File: src/resources.qrc (hunk 1 above)
Comment (dmitry-zezik) [reply src/resources.qrc:162]: Ok

File: src/qml/components/BusyControl.qml (hunk 2 above)
Comment (fpohtmeh-github) [reply src/qml/components/BusyControl.qml:6]: fix

Comment (dmitry-zezik) [reply src/qml/components/BusyControl.qml:6]: No

**Comment by fpohtmeh-github** [comment]
Closing because we decided to display progress in another way.
However, I don't delete the branch

**Pull request closed without merging**